examples/combined_ODNP.py

```diff
@@ -9,10 +9,12 @@
 """
 import numpy as np
 from numpy import r_
-from pyspecdata import *
+import pyspecdata as psd
 from pyspecdata.file_saving.hdf_save_dict_to_group import hdf_save_dict_to_group
-from pyspecdata import strm
-import os, sys, time
+from pyspecdata import strm, DNP_data
```

Comment:
I don't think this would even work. I though DNP_data was something in pyspecdata from the flake assertions but I'm realizing part of my confusion was where the variable is initialized vs where its first used